spec/models/pageflow/home_button_spec.rb

```diff
@@ -5,15 +5,17 @@ module Pageflow
     describe '#url' do
       it 'returns home_url of revision' do
         revision = Revision.new(home_url: 'http://example.com')
-        theming = Theming.new
+        theming = Theming.new(account: create(:account))
```

Comment:
Using the `:theming` factory would be easier since it would automatically create the account.